ksqldb-engine/src/main/java/io/confluent/ksql/statement/ConfiguredStatement.java

```diff
@@ -67,6 +67,15 @@ public String getStatementText() {
     return statement.getStatementText();
   }
 
+  /**
+     * Use masked statement for logging and other output places it could be read by human. It
```

Comment:
Thanks for adding this doc string! Can we also add one for `getStatementText()` above to make extra clear that `getStatementText()` is unsafe for logging / other human-readable usages?

Also, can we update the `toString()` method of this class to use the masked version instead?